src/components/composer/standalone_composer/standalone_composer_store.ts

```diff
@@ -0,0 +1,33 @@
+import { Get } from "../../../store_engine";
+import { CellPosition } from "../../../types";
+import { AbstractComposerStore } from "../composer/composer_store";
+
+export interface StandaloneComposerArgs {
+  onConfirm: (content: string) => void;
+  content: string;
+}
+
+export class StandaloneComposerStore extends AbstractComposerStore {
+  constructor(get: Get, private args: StandaloneComposerArgs) {
+    super(get);
+    this._currentContent = args.content;
+  }
+
+  protected getComposerContent(): string {
+    return this._currentContent;
+  }
+
+  protected getComposerStartingPosition(): CellPosition {
+    return { row: 0, col: 0, sheetId: this.getters.getActiveSheetId() };
+  }
+
+  stopEdition() {
+    this._stopEdition();
+    this.colorIndexByRange = {};
```

Comment:
Shouldn't `this.colorIndexByRange = {};` be managed by the parent class in `_stopEdition` ?
The pattern
```js
this._stopEdition();
this.colorIndexByRange = {};
```
is repeated 2 out of the 3 uses of `_stopEdition`. And no test breaks when adding `this.colorIndexByRange = {};` at the third place